alert-common/src/main/java/com/synopsys/integration/alert/common/action/ValidationActionResponse.java

```diff
@@ -22,12 +22,32 @@
  */
 package com.synopsys.integration.alert.common.action;
```

Comment:
I'm thinking that when we further separate Alert into sub-projects, the models for the REST API stuff should be moved into that sub-project as well. That way we can avoid misusing the models and bogging down alert-common.

Reply:
I think after this PR and the feature branch are merged into master we will no longer need to use AlertFieldException and IntegrationRestException in Alert.  So the static method in this ValidationActionResponse will be able to be removed also.